sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -1,7 +1,6 @@
 SUMMARY="An atomic memory update operations portable implementation"
-DESCRIPTION="Semi-portable access to hardware-provided \
-atomic memory update operations on a number architectures."
-HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
```

Comment:
Oops, what I had in mind was:
```
DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
operations on a number architectures." 
```
I like to use diffs (with red and green lines) but I don't know why I did not use it to comment about `DESCRIPTION`.
Well, I guess I was affraid of being too picky... :D

Anyway, thanks for the updates so far.
(I'll test the recipe asap.)